akka-stream/src/main/resources/reference.conf

```diff
@@ -40,6 +40,17 @@ akka {
 
       # Maximum number of elements emitted in batch if downstream signals large demand
       output-burst-limit = 1000
+
+      interpreter {
```

Comment:
you are right, maybe it should be a "debug" section instead. (could be pulled up one section of course instead of debug, but I would like to isolate this section from prod settings more)